### PR TITLE
Added libassh implementation

### PR DESCRIPTION
**Pull request by diaxen**

### CODE REVIEW

File: _impls/libassh.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+title: libassh
+homepage: http://libassh.org/
+source-repository: https://hg.savannah.nongnu.org/hgweb/libassh/
+license: "[LGPL 2.1](https://git.libssh.org/projects/libssh.git/tree/COPYING)"
```
Comment (fingolfin) [inline _impls/libassh.md]: This link is wrong. Perhaps like this?
```suggestion
license: "[LGPL 2.1](https://www.nongnu.org/libassh/manual/Copyright_and_license.html)"

```

File: _impls/libassh.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+latest-release:
+    version: 1.0
+    date: 2021-01-03
+#changelog: TODO
+client: yes
+server: yes
+library: both
+protocols:
+    cipher:
+	- 3des-cbc
```
Comment (fingolfin) [inline _impls/libassh.md]: Could you please change the tabs to spaces?

File: _impls/libassh.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+title: libassh
+homepage: http://libassh.org/
+source-repository: https://hg.savannah.nongnu.org/hgweb/libassh/
+license: "[LGPL 2.1](https://git.libssh.org/projects/libssh.git/tree/COPYING)"
+first-release:
+    date: 2020-09-28
+latest-release:
+    version: 1.0
+    date: 2021-01-03
+#changelog: TODO
```
Comment (fingolfin) [inline _impls/libassh.md]: No changelog? That's a pity; it's super helpful for all kinds of things (like... "Wait, you are using version 1.0? Oh but support for protocol XYZ was only added in version 1.2")

**Review by fingolfin — COMMENTED** [review]
Thank you very much for your contribution. I have a few minor remarks / requests which you perhaps would be inclined to consider?

File: _impls/libassh.md (hunk 1 above)
Comment (diaxen) [reply _impls/libassh.md]: Thanks. This is fixed in 6dfde26.

File: _impls/libassh.md (hunk 2 above)
Comment (diaxen) [reply _impls/libassh.md]: I changed to spaces in 6dfde26.

File: _impls/libassh.md (hunk 3 above)
Comment (diaxen) [reply _impls/libassh.md]: Yes, I will add a changelog in the next release of the library. There were no protocol change between rc1 and 1.0.

**Review by fingolfin — APPROVED** [review]
Thanks!!!

**Pull request merged**